ex2.py

```diff
@@ -0,0 +1,6 @@
+def get_positive_numbers():
+    """ gets from the user list of numbers (as str) conver it to int and insert it to a list,
+    run on the list and keeps only positive numbers"""
+    numbers=[int(x) for x in input("enter a list of numbers : ").split(",")]
```

Comment:
In this case the assignment was unclear but in general we would  like to seperate IO from method logic
https://wiki.c2.com/?SeparateIoFromCalculation